xbmc/video/Teletext.cpp

```diff
@@ -1303,6 +1304,9 @@ void CTeletextDecoder::RenderPage()
 
 void CTeletextDecoder::DoFlashing(int startrow)
 {
+  if((void *)m_RenderInfo.PageInfo != (void *)m_txtCache->astCachetable[m_txtCache->Page][m_txtCache->SubPage])
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting


> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types


> New code has to use C++ style casts and not older C style casts. [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#122-casts